libs/utils/src/filenames.ts

```diff
@@ -171,7 +153,10 @@ export function parseCastVoteRecordReportDirectoryName(
   // extract number of ballots
   assert(typeof postTestPrefixSegments[1] !== 'undefined');
   const ballotSegments = postTestPrefixSegments[1].split(SUBSECTION_SEPARATOR);
-  if (ballotSegments.length !== 2 || ballotSegments[1] !== 'ballots') {
+  if (
+    ballotSegments.length !== 2 ||
+    (ballotSegments[1] !== 'ballots' && ballotSegments[1] !== 'ballot')
```

Comment:
Bug fix.

Reply:
Regression test?

Reply:
added in 045ff193fd7d6fd6db767acf33180f182970fcfe